service/storage_proxy.cc

```diff
@@ -3813,18 +3813,21 @@ class abstract_read_executor : public enable_shared_from_this<abstract_read_exec
             }
 
             // Waited on indirectly.
-            (void)digest_resolver->done().then(utils::result_into_future<result<>>).then([exec, digest_resolver, timeout, background_repair_check] () mutable {
+            (void)utils::result_futurize_try([&] {
```

Comment:
Do we need to use result_futurize_try to ignore all exceptions? Does `then_wrapped([]{})` will do it?

Reply:
I think this will work:

```c++
.then_wrapped([] (future<result<>>&& f) {
    f.ignore_ready_future();
});
```

Reply:
Done